aquamonitor.py

```diff
@@ -687,6 +686,7 @@ def get_project_chemistry(proj_id, st_dt, end_dt, token=None, n_jobs=None):
 
 def long_to_wide(df_long):
     """ Converts into a wide format df from get_project_chemistry_input """
+    units = df_long[['parameter_name', 'method_unit']].drop_duplicates()

```

Comment:
It seems a little strange for me how you mix parameter and method. There is a naming convention in Nivadatabase that input values are related to a method_id and the method_definitions table. Parameter's is something else and are related to that other function get_project_chemistry. Here you seems to be mixing those two.

Reply:
hm. ok, I am a bit confused now, what is the difference between  'Method.Unit': 'method_unit',
 "Parameter.Unit": "parameter_unit" ?

Reply:
Method.Unit comes from the table method_definitions.unit, while Parameter.Unit comes from the table wc_parameter_definitions.unit. There are procedures to convert from method.name / method.unit to parameter.name / parameter.unit, and the outcome of that procedure is what we get when calling get_project_chemistry.